Client/Application/AppDelegate.swift

```diff
@@ -31,15 +31,25 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         let controller = BrowserViewController()
         controller.profile = profile
+
+        // Add restoration class, the factory that will return the ViewController we 
+        // will restore with.
+        controller.restorationIdentifier = NSStringFromClass(BrowserViewController.self)
+        controller.restorationClass = AppDelegate.self
+
         self.window!.rootViewController = controller
         self.window!.backgroundColor = UIColor(red: 0.21, green: 0.23, blue: 0.25, alpha: 1)
-        self.window!.makeKeyAndVisible()
+
 
 #if MOZ_CHANNEL_AURORA
```

Comment:
I think this Aurora specific code needs to move to `didFinishLaunching:` because it triggers a dialog.